__tests__/app.test.js

```diff
@@ -402,4 +402,93 @@ describe ("DELETE /api/comments/:comment_id", () => {
             })        
         })
       }); 
-  });
\ No newline at end of file
+  });
+
+  //Task 11 CORE: GET /api/articles
+  // Refer to Line 90 test (Task 5), if not specified topic for filtering ,should return all the articles an its default sort_by and order
+  describe ('GET /api/articles', () => {
+    // filters by topic with default sort_by and order 
+    test ('200:responses articles filters by the topic,default sort by date and order by desc', () => { 
+        return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({body}) => {
+              const {articles} = body;  
+               expect(articles).toHaveLength(4);
+               expect(articles).toBeSortedBy("created_at",{descending: true,});
+               articles.forEach((article) => {
+                expect(article).toHaveProperty("author", expect.any(String));
+                expect(article).toHaveProperty("title", expect.any(String));
+                expect(article).toHaveProperty("article_id", expect.any(Number));
+                expect(article).toHaveProperty("topic", 'mitch');  
+                expect(article).toHaveProperty("created_at");    
+                expect(article).toHaveProperty("votes", expect.any(Number));   
+                expect(article).toHaveProperty("article_img_url", expect.any(String));
+                expect(article).toHaveProperty("comment_count", expect.any(Number));
+                expect(article).not.toHaveProperty("body"); 
+            })        
+        })
+      }); 
+ 
+      //validate the topic
+      test ('200:should return empty array if topic was found in topics ,but articles have not this topic',() => {
+        return request(app)
+        .get("/api/articles?topic=paper")
+        .expect(200)
+        .then (({body}) => {
+            const {articles} = body;
+            expect(articles).toHaveLength(0); 
```

Comment:
just add an another expect for an empty array. the toHaveLength matcher can used for a string as well